src/index.js

```diff
@@ -419,14 +419,20 @@ export default React.createClass({
               <span key={col.id}
                 style={{display: 'flex', alignContent: 'flex-end', flex: 1}}>
                 {!col.hideFilter ? (
-                  <input type='text'
-                    style={{
-                      flex: 1,
-                      width: 20
-                    }}
-                    value={filter ? filter.value : ''}
-                    onChange={(event) => this.filterColumn(column, event, col)}
-                  />
+                  _.normalizeComponent(col.filterRender,
+                    {
+                      onFilterChange: (value) => (this.filterColumn(column, value, col))
+                    },
+                    (
+                      <input type='text'
```

Comment:
We should extract this out into a default prop (which will also involve passing more props to it in addition to `onFilterChange`